_rules/text-contrast-afw4f7.md

```diff
@@ -42,7 +42,7 @@ Any [visible][] character in a [text node][] that is a [child](https://dom.spec.
 
 ## Expectation
 
-For each test target, the [highest possible contrast](#highest-possible-contrast) between the [foreground colors](#foreground-colors-of-text) and [background colors](#background-colors-of-text) is at least 4.5:1 or 3.0:1 for [larger scale text](#large-scale-text), except if the test target is part of a [text node][] that is [decorative](#decorative), or does not express anything in [human language](https://www.w3.org/TR/WCAG21/#dfn-human-language-s).
+For each test target, the [highest possible contrast](#highest-possible-contrast) between the [foreground colors](#foreground-colors-of-text) and [background colors](#background-colors-of-text) is at least 4.5:1 or 3.0:1 for [larger scale text](#large-scale-text), except if the test target is part of a [text node][] that is [decorative](#marked-as-decorative), or does not express anything in [human language](https://www.w3.org/TR/WCAG21/#dfn-human-language-s).
```

Comment:
Either use "text node that is [marked as decorative]" or "text node that is [pure decoration] (link to WCAG def)".

And this does change expectation and should probably be part of another PR.

Also, how come this could have been merged without failing a test? As far as I can see, the removal of the definition of "decorative" was done before the merge of the contrast rule. So the later should have failed a test that all defs exist. There's something wrong in our tests.

Reply:
Will wait for @WilcoFiers to resolve this in a different PR and then I can up update this branch.

Reply:
Then, please remove the change from this PR.

Reply:
When Wilco makes his change and when I update this branch, the change will be removed by thyself :).

Reply:
I will make this as WIP for peace of mind until then.